docs/index.md

```diff
@@ -1960,7 +1963,7 @@ source | object | Properties of the source of the payment.
 *source.* maxAmount | [laxAmount](#amount) | The maximum amount to send. (This field is exclusive with source.amount)
 destination | object | Properties of the destination of the payment.
 *destination.* address | [address](#address) | The address to receive at.
-*destination.* amount | [laxAmount](#amount) | An exact amount to deliver to the recipient. If the counterparty is not specified, amounts with any counterparty may be used. (This field is exclusive with destination.minAmount).
+*destination.* amount | [laxAmount](#amount) | *Optional* An exact amount to deliver to the recipient. If the counterparty is not specified, amounts with any counterparty may be used. (This field is exclusive with destination.minAmount.) When parsing a partial payment, this field may be hidden; for the amount that the transaction delivered, see 'deliveredAmount'.
```

Comment:
I understand why the `When parsing a partial payment...` is showing up here, but I think it's a little confusing since it's not relevant to `getPaths`

Reply:
Yeah, I might need to create separate schemas for a transaction request vs. a transaction response.